src/odemis/acq/stream/_sync.py

```diff
@@ -1735,13 +1742,17 @@ def _runAcquisitionScanStage(self, future):
                     if self._acq_state == CANCELLED:
                         raise CancelledError()
 
-                    # MD_POS default to the center of the sample stage, but it
-                    # needs to be the position of the
-                    # TODO: here different from _runAcquisitionEbeam
-                    # sample stage + e-beam + scan stage translation (without the drift cor)
+                    # MD_POS default to the center of the sample stage, but it needs to be the position
+                    # of the sample stage + e-beam + scan stage translation (without the drift cor)
+                    # TODO: here the code is different compared to _runAcquisitionEbeam
                     raw_pos = self._acq_data[0][-1].metadata[MD_POS]
                     strans = spos[0] - orig_spos["x"], spos[1] - orig_spos["y"]
-                    cor_pos = raw_pos[0] + strans[0], raw_pos[1] + strans[1]
+                    # check if a scan-stage wrapper (has stage dependency) is used
+                    # if so only use the translated position and don't add to raw
```

Comment:
Check whether the scan stage is just the standard stage (ie, the scan-stage component "affects" the stage),
or it's a completely independent stage (in which case the "raw" MD_POS must be added to the translation.